pkg/kudoctl/cmd/init.go

```diff
@@ -98,6 +99,7 @@ func newInitCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 	f.BoolVar(&i.crdOnly, "crd-only", false, "Add only KUDO CRDs to your cluster")
 	f.BoolVarP(&i.wait, "wait", "w", false, "Block until KUDO manager is running and ready to receive requests")
 	f.Int64Var(&i.timeout, "wait-timeout", 300, "Wait timeout to be used")
+	f.StringVar(&i.webhooks, "webhook", "", "List of webhooks exposed, when empty, no webhook server will be started (the only webhook right now is InstanceValidation)")
```

Comment:
```suggestion
	f.StringVar(&i.webhooks, "webhook", "", "List of webhooks to install separated by commas")
```
"list of webhooks exposed" isn't phrased corrected.  "to be exposed" is closer.   It is odd to add the "(...)" to a cli output or to specify limitations that will need to be edited and hopefully not forgotten.

The info I expect a user to be concerned with... is... is it the name of the class? does it include the package name?    where can I find the mapping of strings -> webhooks.

Reply:
hmm I really wanted to help the user to know what to do here because if you know what are the supported options you have to go to the docs or code (?) and that's not very nice experience. I think the CLI should help you do this right. I rephrased it a bit.

Yes we will need to update it with every new webhook but this is the best documentation we have and the best documentation we can get to the user so I think we should be fine with it and just really take care of this docs and keep it up to date. We won't certainly add new webhook every week...